### ISSUE TITLE
Write tests for botspade

### ISSUE THREAD

**Issue by jasonp** [jasonp/botspade#11]
Getting complex enough that it needs tests. What to test?

bot responds to !points
bot responds to !uptime
bot accepts correctly formatted !bet 
bot rejects incorrectly formatted !bet
bot responds to !stats
bot does !update profile
bot does !lookup profile/index
bot does !checkin if more than 12 hours have passed
bot rejects !checkin if fewer than 12 hours have passed
etc,

so:
if bot receives !points
  db retrieves user, points displayed

if bot receives !bet 15 win
  bot IDs user and
  bot records 15 point bet

etc

**Comment by jasonp** [comment]
I am especially lost on this one since the bot isn't a CLASS, which it seems like it's easy to write unit and spec tests for CLASSES in ruby using minitest... I have no idea how to test the activities of a bunch of non-object oriented code in a script, basically. 

Even if we instantiated a bot using the method described in the isaac documentation, we still would have only an instance not a class, and I'm not sure how to create a test instance of an instance, basically, and beyond that, I'm not sure how to pass input to it the way IRC does. 

IF we got THAT far, we could then just test for assert_equal the output we want, and that'd be _awesome,_ but there's a lot I don't know to begin with.
